stream/src/test/java/org/apache/kafka/test/MockTimestampExtractor.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kafka.test;
+
+import org.apache.kafka.streaming.processor.TimestampExtractor;
+
+public class MockTimestampExtractor implements TimestampExtractor {
+
+    @Override
+    public long extract(String topic, Object key, Object value) {
+        return ((Integer) key).longValue();
```

Comment:
Why not use ``((Long) key);`` or even ``((Integer) key)``. No need to call ``longValue()`` as the autoboxing is called automatically, afaik.